receiver/filestatsreceiver/metadata.yaml

```
@@ -57,5 +57,11 @@ metrics:
     gauge:
       value_type: int
     unit: "b"
+  file.count:
+    description: The number of files matched
+    enabled: false
+    gauge:
+      value_type: int
+    unit: "{files}"
```

OTel spec recommends using singular form
```suggestion
    unit: "{file}"
```